testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpServerTest.groovy

```diff
@@ -602,6 +602,9 @@ abstract class HttpServerTest<SERVER> extends InstrumentationSpecification imple
         }
       }
       attributes {
+        if (extraAttributes.contains(SemanticAttributes.NET_PEER_NAME)) {
+          "${SemanticAttributes.NET_PEER_NAME}" address.host
+        }
```

Comment:
I ran tests and this is needed to pass on windows anyways, and would eliminate needing to add `NET_PEER_NAME` to extraAttributes
```suggestion
        "${SemanticAttributes.NET_PEER_NAME}" { it == null || it == address.host }
```